modules/hostap/src/supp_api.h

```diff
@@ -73,6 +73,14 @@ int supplicant_scan(const struct device *dev, struct wifi_scan_params *params,
 int supplicant_get_stats(const struct device *dev, struct net_stats_wifi *stats);
 #endif /* CONFIG_NET_STATISTICS_WIFI || __DOXYGEN__ */
 
+/** Flush PMKSA cache entries
+ *
+ * @param dev Pointer to the device structure for the driver instance.
+ *
+ * @return 0 if ok, < 0 if error
+ */
+int supplicant_pmksa_flush(const struct device *dev);
```

Comment:
Same, please add how/where is this used?

Reply:
It's used in NXP wifi code, and user can decide to use this API to Flush PMKSA cache entries

Reply:
it would be better to add this to Wi-Fi shell so that we can test/see the usage of the API.

Reply:
Hi @krish2718 can I only add this into wifi_mgmt.c?